desc/equilibrium/coords.py

```diff
@@ -110,6 +110,7 @@ def periodic(x):
     # do surface average to get iota once
     if "iota" in profiles and profiles["iota"] is None:
         profiles["iota"] = eq.get_profile("iota", params=params)
+        params["i_l"] = profiles["iota"].params
```

Comment:
can you explain how this fixes it?

Reply:
```python
def _iota_num_current(params, transforms, profiles, data, **kwargs):
    """Current contribution to the numerator of rotational transform formula."""
    if profiles["iota"] is not None:
        # otherwise params is empty
        iota = profiles["iota"].compute(transforms["grid"], params["i_l"], dr=0)
        ..... # above function eventually calls the following line with f = params = empty list which errors
        fq = interp1d(xq, x, f, method=self._method, derivative=dr, extrap=True)
```